kv/test_raftstore/test_test.go

```diff
@@ -98,28 +98,31 @@ func checkConcurrentAppends(t *testing.T, v string, counts []int) {
 }
 
 // repartition the servers periodically
-func partitioner(t *testing.T, cluster *Cluster, ch chan bool, done *int32, unreliable bool, electionTimeout time.Duration) {
+func partitioner(t *testing.T, cluster *Cluster, ch chan bool, done *int32, unreliable bool, partitions bool, electionTimeout time.Duration) {
```

Comment:
how about renaming the `partitioner` to `networkchaos` or something else

Reply:
Sorry to resp so late, your advice souds great.